pex/cli/commands/lock.py

```diff
@@ -289,17 +288,23 @@ def _dump_lockfile(
     ):
         # type: (...) -> None
         path_mappings = resolver_options.get_path_mappings(self.options)
-        dump = functools.partial(
-            self.dump_json,
-            self.options,
-            json_codec.as_json_data(lockfile=lock_file, path_mappings=path_mappings),
-            sort_keys=True,
-        )
+        def dump_with_terminating_newline(out):
```

Comment:
CI is failing for lack of a newline between `path_mappings = ...` and `def ...`.